README.rst

```diff
@@ -0,0 +1,168 @@
+.. Copyright 2020-2022 Robert Bosch GmbH
```

Comment:
Hi Ngoan,
please remove the licence header here.
The licence at the bottom is enough.
Thank you,
 Thomas

Reply:
Hi Thomas,

I have removed it and fixed some issue in README file in new commit.

Thank you,
Ngoan